src/handlePreviewToggles.ts

```diff
@@ -1,36 +1,39 @@
 "use strict";
 
-import { unset, has } from "lodash";
 import { bold, red } from "colorette";
 
-import { configstore } from "./configstore";
-import { previews } from "./previews";
+import * as experiments from "./experiments";
 
-function _errorOut(name?: string) {
-  console.log(bold(red("Error:")), "Did not recognize preview feature", bold(name || ""));
+function errorOut(name?: string): void {
+  console.log(`${bold(red("Error:"))} Did not recognize preview feature ${bold(name || "")}`);
   process.exit(1);
 }
 
-export function handlePreviewToggles(args: string[]) {
-  const isValidPreview = has(previews, args[1]);
+/**
+ * Implement --open-sesame and --close-sesame
+ */
+export function handlePreviewToggles(args: string[]): boolean {
```

Comment:
Does this _need_ to return a boolean? Looks like it could just be `: void`

Reply:
The function returned falsey (undefined) when neither command was used, and this was used in if statements elsewhere. I formalized it by making it boolean and only returning false (or exiting)